style.css

```diff
@@ -71,38 +88,63 @@ nav a {
 .aboutMe_section {
 	display: flex;
   flex-direction: column;
-/*	align-items: center;*/
+	align-items: flex-start;
+	justify-content: center;
+	background-color: #262323;
+	padding-top: 50px;
+	padding-bottom: 50px;
+	
 }
 
 .aboutMe_card {
 	display: flex;
+	flex-direction: row;
 	flex-wrap: wrap;
 	justify-content: flex-start;
+	align-items: center;
 	margin-left: 50px;
 	gap: 20px;
 }
 
 .aboutMe_section img {
 	width: 200px;
+	height: 250px;*/
 }
 
-
 .aboutMe_section h2 {
 		margin-left: 50px;
 	}
 
 .aboutMe_text {
 	display: flex;
 	flex-direction: column;
-	align-items: center;		
-	max-width: 500px;
-	
+	align-items: flex-start ;		
+	max-width: 1000px;
+	gap: 20px;
+}
+
+.aboutMe_text p, .aboutMe_text a {
+	font-size: 24px;
+}
+
+.aboutMe_text a {
+	color: white;
+	background-color: #d65a64;
+	padding: 5px;
+	border-radius: 10px;
 }
 
 /* Portfolio section  */ 
 
+.portfolio_section {
+	background-color: #171616;
+/*  	padding: 50px 50px;*/
+	padding-top: 50px;
+	padding-bottom: 50px;
+}
+
 .portfolio_section h2 {
-	margin-left: 50px;
+	margin: 0;
```

Comment:
also some margin-left? 
or padding-left in .portfolio_section?
same as with introduction text...

Reply:
done